### PR TITLE
Ausops Turfs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Work in progress I guess, the details aren't set, but functional enough to start with.

learntoembedkor

### CODE REVIEW

**Comment by Jordie0608** [comment]
> Sprite PR without pictures

I never thought this would become a recurring thing

**Comment by Shadowlight213** [comment]
Kek

**Comment by Ausops** [comment]
<blockquote class="imgur-embed-pub" lang="en" data-id="M9WLwnq"><a href="//imgur.com/M9WLwnq">View post on imgur.com</a></blockquote><script async src="//s.imgur.com/min/embed.js" charset="utf-8"></script>

**Comment by TechnoAlchemisto** [comment]
wow that actually looks pretty damn good my dude

**Comment by Jazaen** [comment]
Good sprites in SS13, not limited to some alien-snowflake (LOOKING AT YOU, XENOS AND ABDUCTORS)? Welp, better prepare myself for the end of the world.

**Comment by RemieRichards** [comment]
These are nice

**Comment by xxalpha** [comment]
I don't like the walls and the floor that looks like the walls, and what is that blue shit?

The rest is pretty good.

**Comment by Dorsisdwarf** [comment]
This could be a peaceful mining world in comparison to Lavaland Hellhole

**Comment by Ausops** [comment]
savage monsters that want nothing more than to eat you in the same way a praying mantis slowly eats away at the face of a bee are on the way

File: code/modules/mining/mine_turfs.dm [hunk 1]
```diff
@@ -693,3 +692,46 @@
...
+	smooth = SMOOTH_MORE|SMOOTH_BORDER
+	canSmoothWith = list (/turf/closed/mineral, /turf/closed/wall)
+	baseturf = /turf/open/floor/plating/ash
+	initial_gas_mix = "o2=14;n2=23;TEMP=300"
+	environment_type = "waste"
+	turf_type = /turf/open/floor/plating/ash
+
+/turf/open/floor/plating/ash
+	pixel_y = -4
+	pixel_x = -4
```
Comment (ChangelingRain) [inline code/modules/mining/mine_turfs.dm]: You need to set this and the icon in New() so it's easy for mappers to place it

File: code/modules/mining/mine_turfs.dm [hunk 2]
```diff
@@ -693,3 +692,46 @@
 	turf_type = /turf/open/floor/plating/asteroid/basalt/lava_land_surface
 	baseturf = /turf/open/floor/plating/asteroid/basalt/lava_land_surface
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
+
+
+//BECAUSE ONE PLANET WASNT ENOUGH
+
+/turf/closed/mineral/ash_rock //wall piece
+	pixel_y = -4
+	pixel_x = -4
```
Comment (ChangelingRain) [inline code/modules/mining/mine_turfs.dm]: Same with this, etc

File: code/modules/mining/mine_turfs.dm [hunk 3]
```diff
@@ -25,7 +25,6 @@
 /turf/closed/mineral/New()
 	pixel_y = -4
 	pixel_x = -4
-	icon = 'icons/turf/smoothrocks.dmi'
```
Comment (ChangelingRain) [inline code/modules/mining/mine_turfs.dm]: This is necessary so that placing it in the map editor works properly.

**Comment by ChangelingRain** [comment]
Looks worryingly fungal 10/10
I await miners returning horrible infections to the station

**Comment by MMMiracles** [comment]
slowly but surely, everything that isn't the station will look better than the station itself.

**Comment by Cheridan** [comment]
And then we remove the station and repackage the game as Lavaland Adventures: The Minening to avoid licensing issues and make mad $$$$$$$$$$

**Comment by TechnoAlchemisto** [comment]
Mushroom land

**Comment by duncathan** [comment]
Not dark enough. The dark atmosphere of lavaland is wonderful

**Comment by KorPhaeron** [comment]
@duncathan 

I think it's too dark to have the station be on. It'd be too grim to spend all our time there.

Which is why this will be the surface, with the station on it, and lavaland will be underneath

**Comment by duncathan** [comment]
so lavaland was jungle station all along

**Comment by KorPhaeron** [comment]
I've been startoad all along

**Comment by Jazaen** [comment]
So... This all is... Just as Planned?
Huh.
I guess it was outlined in the """"DESIGN"""" document, but I never believed that any steps would be made in this direction.

File: code/modules/mining/mine_turfs.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/modules/mining/mine_turfs.dm]: Fiiiiiiine

**Comment by KorPhaeron** [comment]
@RemieRichards @ChangelingRain  @xxalpha 

There has got to be a less hacky way of using smooth turfs on maps than having duplicate icons in another file, then switching the icon file and pixel shift on New()

**Comment by ChangelingRain** [comment]
That /is/ the least hacky way.

**Comment by xxalpha** [comment]
What the fuck do you mean duplicate icons? What is the problem here?

**Comment by KorPhaeron** [comment]
@xxalpha 

Joan put the nonsmooth rock icons in one file (which looks the same as the complete version of the smoothed icon), then changes the icon to a new file full of smoothed stuff on New()

**Comment by xxalpha** [comment]
PUT THAT ICON IN THE SMOOTH ICON FILE AND SET THE ICON STATE TO THAT ICON AND THEN SHIT HANDLES ITSELF

**Comment by KorPhaeron** [comment]
No but the issue is that the turfs are bigger than 32x32, so they look weird in the map editor without the pixel offset since they're not centered

Maybe I'll just do it the hacky way

**Comment by ChangelingRain** [comment]
That doesn't work because the icon file for smoothrocks(and these) is 40x40 so byond fucks it up.
It's not hacky, because there's no other way to do it.

**Comment by Dorsisdwarf** [comment]
What why

**Comment by TechnoAlchemisto** [comment]
:(

**Comment by Jordie0608** [comment]
It was reopened #17603

**Pull request closed without merging**